tasks/main.yml

```diff
@@ -8,18 +8,19 @@
     dest: /etc/localtime
     state: link
     force: yes
+  when: ntp_enabled
```

Comment:
I'd also like to remove these extra conditions—NTP should be installed even if not enabled. If the installation starts the service by default (I'm looking at you, weird Debian packages), then that's not our fault, but I'd rather stick with package defaults and make sure it's still installed at least. If you don't want it installed, don't include the role—e.g. in your playbook, add:

```
  roles:
    - role: geerlingguy.ntp
      when: ntp_enabled
```